kubernetes/src/test/java/com/linecorp/armeria/client/kubernetes/endpoints/KubernetesEndpointGroupMockServerTest.java

```diff
@@ -77,6 +78,11 @@ static void afterAll() {
         staticClient.close();
     }
 
+    @AfterEach
+    void afterEach() {
+        client.close();
```

Comment:
Does this prevent a leak reported in CI? AFAIK, an instance client is correctly closed by `KubernetesMockServerExtension`.

Reply:
Not sure, I thought I saw a stacktrace but the reference in the pr description seems to be wrong. Let me just revert.